packages/gguf/src/gguf.ts

```diff
@@ -308,6 +308,9 @@ export async function gguf(
 			}
 		}
 		offset += valueResult.length;
+		/// TODO(fix typing)
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore
```

Comment:
Let's use `// @ts-expect-error` instead of `// @ts-ignore` in general

(no need for eslint-disable this way)

Here I think you can change `const metadata: GGUFMetadata` to `const metadata: GGUFMetadata<GGUFType.NON_STRICT>` to remove the error (not sure if it's the best fix)

Reply:
Fixed in https://github.com/huggingface/huggingface.js/pull/655/commits/31bac8bbba365e13c517c48452fdee1d5843abd3